gems/pending/metadata/util/win32/Win32EventLog.rb

```diff
@@ -525,7 +525,7 @@ def addNodeRec(node, rec)
     # Put the needed fields in the node_rec, and collect them for md5 hashing to
     #   verify that this record is unique
     md5 = NODE_REC_KEYS.collect { |k| node_rec[k] = rec[k] }.join(' ')
-    md5 = Digest::MD5.hexdigest(md5)
+    md5 = Rails.application.config.digest_class.hexdigest(md5)
```

Comment:
@gmcculloug or @roliveri Please review.  Would it be possible to use an alternative to md5 here?